CONTRIBUTING.md

```diff
@@ -54,8 +54,11 @@ Thank you! We'll try to get back to you as soon as possible.
 
 1. Install Rust using [rustup], which allows you to easily switch between Rust
    versions.
+
+   Diesel currently supports Rust Stable & Rust Nightly, but does not guarantee support for Rust Beta.
```

Comment:
This is incorrect. We support stable, beta, and nightly. There's an issue with our doctests failing the beta build at this moment, but that doesn't affect our support.